gym_demo.py

```diff
@@ -4,14 +4,21 @@
 observation = env.reset()
 done = False
 score = 0
+
+print('Action space: ', env.action_space)
+print(env.unwrapped.get_action_meanings())
+print('Observation space: ', env.observation_space)
+
 while not done:
-    # env.render()
-    action = env.action_space.sample(
-    )  # your agent here (this takes random actions)
+    env.render()
```

Comment:
Nie chcemy, by demo było domyślnie renderowane, stwarza to problemy, m.in. wewnątrz dockera.